tools/accuracy_checker/configs/handwritten-japanese-recognition-0001.yml

```diff
@@ -0,0 +1,40 @@
+models:
+  - name: handwritten-japanese-recognition-0001
+
+    launchers:
+      - framework: dlsdk
+        device: cpu
+        tags:
+          - FP16
```

Comment:
@vladimir-dudnik @IRDonch will the model available only in FP16 precision or FP32 version also be places in OMZ?

Reply:
@eaidova @cocokale I did not saw the model itself yet, but I think it definitely have to be available in FP32 precision as well.